src/raw/generic.rs

```diff
@@ -66,23 +68,28 @@ impl Group {
 
     /// Loads a group of bytes starting at the given address.
     #[inline]
-    pub unsafe fn load(ptr: *const u8) -> Group {
-        Group(ptr::read_unaligned(ptr as *const _))
+    #[allow(clippy::cast_ptr_alignment)] // unaligned load
+    pub unsafe fn load(ptr: *const u8) -> Self {
+        Self(ptr::read_unaligned(ptr as *const _))
     }
 
     /// Loads a group of bytes starting at the given address, which must be
     /// aligned to `mem::align_of::<Group>()`.
     #[inline]
-    pub unsafe fn load_aligned(ptr: *const u8) -> Group {
-        debug_assert_eq!(ptr as usize & (mem::align_of::<Group>() - 1), 0);
-        Group(ptr::read(ptr as *const _))
+    #[allow(clippy::cast_ptr_alignment)]
+    pub unsafe fn load_aligned(ptr: *const u8) -> Self {
+        // FIXME: use align_offset once it stabilizes
```

Comment:
What do we need `align_offset` for? The current code seems to work fine in miri.

Reply:
Align offset is the std lib function I always use for this, but it's unstable.